provider/ansible.py

```diff
@@ -163,19 +156,27 @@ def distro_install():
             else:
                 LOG_JOB.error("Ansible repo was required, but failed to be added.")
                 return False
-        install_status = utils_package.package_install('ansible')
+        install_status = utils_package.package_install(packages)
         if not install_status:
-            LOG_JOB.error(f"Failed to install ansbile.")
+            LOG_JOB.error(f"Failed to install {packages}.")
         yum_backend.remove_repo(params["ansible_repo"])
         return install_status
 
     policy_map = {"distro_install": distro_install,
                   "python_install": python_install}
 
+    # Install ansible depended packages that can't be installed
+    # by pip (or are not a dependency) when installing ansible
+    package_list = params.get_list("package_list", 'sshpass')
     try:
         path.find_command('ansible-playbook')
     except path.CmdNotFoundError:
         ansible_install_policy = params['ansible_install_policy']
-        return policy_map[ansible_install_policy]()
-    else:
-        return True
+        if ansible_install_policy == 'distro_install':
+            package_list.append('ansible')
+        else:
+            if not policy_map[ansible_install_policy]():
+                return False
+
```

Comment:
The distro install part was missed,  i guess it is typo :)

Reply:
Ah, it is! And an important one, fixed :wink: